GANDLF/utils/gandlf_logger.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import yaml
+from pathlib import Path
+from importlib import resources
+import colorlog
+
+
+def gandlf_logger_setup(log_dir=None, config_path="logging_config.yaml"):
+    """
+    It sets up the logger. Reads from logging_config.
+    If log_dir is None, the logs are flashed to console.
+    Args:
+        log_dir (str): dir path for saving the logs
+        config_path (str): file path for the configuration
+
+    """
+
+    if log_dir == None:  # flash logs
```

Comment:
Per [pep-8 suggestions](https://peps.python.org/pep-0008/#programming-recommendations):
```suggestion
    if log_dir is None:  # flash logs
```